src/towncrier/_shell.py

```diff
@@ -13,14 +13,25 @@
 
 from click_default_group import DefaultGroup
 
-from ._version import __version__
 from .build import _main as _build_cmd
 from .check import _main as _check_cmd
 from .create import _main as _create_cmd
 
 
+try:
```

Comment:
Maybe we can still keep the `_version.py` file but this time just just a `_version` string variable.

I this way, we don't have to repeat the `importlib` code each time we want to fetch the version

Reply:
We can do this lazily using `towncrier.__version__` if the block that I've commented on earlier is ported and kill two birds with one stone.